### PR TITLE
Update master branch with changes from v2.1.0

### PR DESCRIPTION
**Pull request by facuspagnuolo**

### CODE REVIEW

File: examples/truffle-migrate/README.md [hunk 1]
```diff
@@ -1,14 +1,22 @@
 # Sample ZeppelinOS project with truffle migrations
 
-This is a sample ZeppelinOS project that runs `zos` from truffle migrations, using the CLI javascript interface. This allows any project that currently relies on truffle migrations to switch over to `zos`, thus deploying upgradeable instances instead of regular ones. Also, by using the CLI interface, `zos` keeps track of all deployed logic contracts and all upgradeable instances, and also tracks them in truffle build artifacts.
+This is a sample ZeppelinOS project that runs `zos` from truffle migrations, using the CLI javascript interface.
```
Comment (jcarpanelli) [inline examples/truffle-migrate/README.md]: Have these new lines been added deliberately? I would revert these in favor of not adding new lines after a certain number of words... but as we don't have any convention defined, I'm ok leaving this as it is..

File: packages/cli/src/commands/verify.ts [hunk 2]
```diff
@@ -14,7 +14,7 @@ const register: (program: any) => any = (program) => program
   .option('-o, --optimizer', 'enables optimizer option')
   .option('--optimizer-runs <runs>', 'specify number of runs if optimizer enabled.')
   .option('--remote <remote>', 'specify remote endpoint to use for verification')
-  .option('--api-key <key>', 'specify etherscan API key. To get one, go to: https://etherscan.io/myapikey')
+  .option('--api-key <key>', 'specify etherscan API key. To get one, go to: https://etherscancom.freshdesk.com/support/solutions/articles/35000022163-i-need-an-api-key')
```
Comment (jcarpanelli) [inline packages/cli/src/commands/verify.ts]: These change should be reverted. The correct URL is `https://etherscan.io/myapikey` (as it is in `master`)

File: packages/cli/src/models/compiler/Compiler.ts [hunk 3]
```diff
@@ -7,9 +7,10 @@ const Compiler = {
   async call(): Promise<{ stdout: string, stderr: string }> {
     log.info('Compiling contracts with Truffle...');
     return new Promise((resolve, reject) => {
-      exec('npx truffle compile', (err, stdout, stderr) => {
-        if (err) reject(err);
-        else resolve({ stdout, stderr });
+      exec(`${process.cwd()}/node_modules/.bin/truffle compile`, (error, stdout, stderr) => {
+        if (stdout) console.log(stdout)
```
Comment (jcarpanelli) [inline packages/cli/src/models/compiler/Compiler.ts]: Should we be logging this with `console.log`?

File: packages/cli/src/scripts/verify.ts [hunk 4]
```diff
@@ -2,7 +2,9 @@ import ControllerFor from '../models/network/ControllerFor';
 import { VerifyParams } from './interfaces';
 
 export default async function verify(contractAlias, { network = 'mainnet', txParams = {}, networkFile, optimizer = false, optimizerRuns = 200, remote = 'etherchain', apiKey }: VerifyParams): Promise<void | never> {
-  if (remote === 'etherscan' && !apiKey) throw new Error('Etherscan API key not specified. To get one, follow this link: https://etherscan.io/myapikey');
+  if (remote === 'etherscan' && !apiKey) {
+    throw new Error('Etherscan API key not specified. To get one, follow this link: https://etherscancom.freshdesk.com/support/solutions/articles/35000022163-i-need-an-api-key');
```
Comment (jcarpanelli) [inline packages/cli/src/scripts/verify.ts]: Same as above, revert or change the etherscan API url to https://etherscan.io/myapikey

File: examples/truffle-migrate/README.md (hunk 1 above)
Comment (facuspagnuolo) [reply examples/truffle-migrate/README.md]: No, that was part of an update in v2.1.0

**Pull request merged**